Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/DataSynchronizer.swift

```diff
@@ -906,6 +1116,12 @@ public class DataSynchronizer: NetworkStateDelegate, FatalErrorListener {
         }
     }
 
+    // MARK: Utilities
+
+    private static func documentIdFilter(forId id: BSONValue) -> Document {
```

Comment:
[nit] I intentionally haven't added this method. It's actually more verbose than just writing out the document– would be nice to take it back out.

Reply:
fair enough, getting rid of it.